pkg/crcontrollers/drive/drivecontroller.go

```diff
@@ -229,7 +224,7 @@ func (c *Controller) handleDriveUpdate(ctx context.Context, log *logrus.Entry, d
 			return remove, nil
 		}
 	case apiV1.DriveUsageFailed:
```

Comment:
From failed status we need to be able to move to the REMOVED state.

Reply:
We check and change state to REMOVED before main switch. It works for any state

Reply:
What is the use case?

Reply:
Force disk removing, if disk is empty or it's already OFFLINE. Is it better leave only FAILED -> REMOVED?